source/transfers/contracts/USDTTransferHandler.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity 0.5.12;
+
+import "./interfaces/ITransferHandler.sol";
+import "./interfaces/IUSDTTransferHandler.sol";
+
+contract USDTTransferHandler is ITransferHandler {
```

Comment:
Is a separate handler still needed for USDT, OMG etc? I thought that the use of the SafeERC20 library meant that non-returning tokens were now handled too?

Reply:
no, you're correct. this handler is being removed =)